mettle/src/stdapi/sys/process.c

```diff
@@ -252,9 +252,14 @@ sys_process_execute(struct tlv_handler_ctx *ctx)
 	struct process_options opts = {
 		.process_name = path,
 		.args = args,
-		.flags = PROCESS_CREATE_SUBSHELL
+		.flags = 0
 	};
 
+	if (strchr(path, '$') != NULL || strchr(path, '%') != NULL ||
```

Comment:
maybe it makes more sense to move this check into src/process.c, since it's OS-specific

Reply:
OTOH, I think by this point in the code we know if we need shell/no shell and having the API magically make that decision seems wrong.

Reply:
OK, so it's fine to leave it here then.

Reply:
Looks good to me

Reply:
okey doke, thanks all